tests/test_0755-dask-awkward-column-projection.py

```diff
@@ -0,0 +1,25 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/uproot4/blob/main/LICENSE
+
+import numpy
+import pytest
+import skhep_testdata
+
+import uproot
+
+dask = pytest.importorskip("dask")
+dask_awkward = pytest.importorskip("dask_awkward")
+
+pytest.importorskip("pyarrow")  # dask_awkward.lib.testutils needs pyarrow
```

Comment:
As we talked about in the meeting, this is no longer necessary.

```suggestion
```

Now we'll see if pytest agrees!